pkg/api/short_urls.go

```diff
@@ -0,0 +1,22 @@
+package api
+
+import (
+	"strings"
+
+	"github.com/grafana/grafana/pkg/api/dtos"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/shorturls"
+)
+
+// POST /api/goto
```

Comment:
Documentation comments should be [full sentences](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences).

```suggestion
// CreateShortURL handles requests to create a short URL.
```